log-storage/aws/dynamodb/iam.tf

```diff
@@ -73,6 +80,21 @@ resource "aws_iam_role" "log_writer" {
           Effect   = "Allow"
           Resource = aws_dynamodb_table.deployment_logs_table.arn
         },
+        {
+          Action   = [
```

Comment:
do we need all of them? for example, is update required?

Reply:
We got it for the other logs table (and for our own) so I think we do... but not really sure what it is used for